lib/v2/weather/index.js

```diff
@@ -0,0 +1,36 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const locationUtil = require('./util/location');
+const config = require('@/config').value;
+const rootUrl = 'https://devapi.qweather.com/v7/weather/3d?';
+
+module.exports = async (ctx) => {
+    let id = await ctx.cache.get(ctx.params.location + '_id');
```

Comment:
Any reason why you're using `ctx.cache.get` instead of the recommended `ctx.cache.tryGet`?